### ISSUE TITLE
It looks dose not support New Webhook

### ISSUE THREAD

**Issue by saitotqr** [mailgun/mailgun-go#187]
I'm trying to export mailgun settings to Json. but It looks dose not support New Webhook
version:mailgun-go/v3

When I added `Webhook/Permanent` Failure then `ListWebhooks` method does't work.
Because `permanent_fail` is New Webhook, and they returns returns LIST response.

```
curl -sS -X 'GET' --user 'api:XXXX' 'https://api.mailgun.net/v3/domains/MYDOMAIN/webhooks' 
{
  "webhooks": {
    "drop": {
      "url": "https://MYDOMAIN/my-dropped-endpoint"
    },
    "permanent_fail": {
      "urls": [
        "http://MYDOMAIN/my-permanent-fail-endpoint"
      ]
    }
  }
}
```
note: `drop` is Legacy Webhook, `permanent_fail` is new webhook.

error cause :
webhook.go/ListWebhooks looks doesn't support LIST retrun.
```
goroutine 18 [running]:
testing.tRunner.func1(0xc000112200)
        /usr/local/Cellar/go/1.12.5/libexec/src/testing/testing.go:830 +0x392
panic(0x133f3e0, 0xc00008f410)
        /usr/local/Cellar/go/1.12.5/libexec/src/runtime/panic.go:522 +0x1b5
github.com/mailgun/mailgun-go.(*MailgunImpl).ListWebhooks(0xc0000cc0f0, 0x1419380, 0xc0000a2180, 0x1419380, 0xc0000a2180, 0xc00008ce00)
        /Users/ME/go/src/github.com/mailgun/mailgun-go/webhooks.go:32 +0x36e
_/Users/ME/work/hrmony/mailgun/settings.ListWebhooks(0x13a3054, 0x11, 0xc0000180b0, 0x32, 0x0, 0x0, 0x0)
        /Users/ME/work/hrmony/mailgun/settings/exporter.go:75 +0x141
_/Users/ME/work/hrmony/mailgun/settings.TestListWebhook(0xc000112200)
        /Users/ME/work/hrmony/mailgun/settings/exporter_test.go:77 +0x58
testing.tRunner(0xc000112200, 0x13b51b0)
        /usr/local/Cellar/go/1.12.5/libexec/src/testing/testing.go:865 +0xc0
created by testing.(*T).Run
        /usr/local/Cellar/go/1.12.5/libexec/src/testing/testing.go:916 +0x35a
exit status 2
```

**Comment by thrawn01** [comment]
@saitotqr thank you for reporting this issue, I have a fix ready once it's reviewed I will tag it as `v3.5.0`

**Comment by thrawn01** [comment]
Fix should be in [Release 3.5.0](https://github.com/mailgun/mailgun-go/releases/tag/v3.5.0)
